docs/console/docs/01-01-console.md

```diff
@@ -11,4 +11,14 @@ Use the following CRs to modify the Console UI:
 
 - The MicroFrontend custom resource allows you to plug in micro front-ends for a specific Namespace.
 - The ClusterMicroFrontend custom resource allows you to plug in micro front-ends for the entire Cluster.
-- The BackendModule custom resource allows you to enable UI API Layer modules.
\ No newline at end of file
+- The BackendModule custom resource allows you to enable UI API Layer modules.
+  
+
+#### Security guidelines for `MicroFrontend` and `ClusterMicroFrontend` resources:
+
+We highly recommend to include the following configuration to your `MicroFrontend` or `ClusterMicroFrontend` web server config file:
+ - Make it only accessible through HTTPS.
```

Comment:
```suggestion
 - Make the configuration accessible only through HTTPS.
```

Reply:
What about `Make the MicroFrontend or ClusterMicroFrontend accessible oly through HTTPS`?

Reply:
done